### ISSUE TITLE
Enhancement request - when enabling S3 for alces storage commands on the login node, also enable s3 on compute nodes

### ISSUE THREAD

**Issue by wdm78** [alces-software/flight-aws-marketplace#75]
This is an enhancement request to make s3 back-end automatically enable itself on compute nodes after it's been enabled on the login node. Alternatively, maybe Flight should be launched with S3 already enabled on both login and compute nodes?

This is the output from what currently happens if you enable S3 on the login node then try and use it from a compute node:

```
[alces@login1(gwtest) ~]$ pdsh -g nodes uptime
ip-10-75-0-91:  19:12:28 up 2 min,  0 users,  load average: 0.46, 0.39, 0.16
ip-10-75-0-162:  19:12:28 up 2 min,  0 users,  load average: 0.80, 0.64, 0.27
[alces@login1(gwtest) ~]$ alces storage enable s3
  >>> alces service install: service already installed: s3cmd
alces storage enable: enabled storage type: base/s3 -> s3
[alces@login1(gwtest) ~]$ alces storage configure mys3 s3
Display name [mys3]:
Access key: XXXXXXXXXXXXXXXXXXXX
Secret key: XXXXXXXXXXXXXXXXXXXXXXXX
Service address [s3.amazonaws.com]:
alces storage configure: storage configuration complete
[alces@login1(gwtest) ~]$ alces storage use mys3
alces storage use: storage configuration 'mys3' now set as default
[alces@login1(gwtest) ~]$ alces storage ls s3://jobscripts
                       DIR   s3://jobscripts/jobdata/
                       DIR   s3://jobscripts/scripts/
2016-05-21 16:08  65536000   s3://jobscripts/.
2016-05-29 16:36    149853   s3://jobscripts/LC82030242016125LGN00.jpg
2016-05-29 16:23      1055   s3://jobscripts/tropic_cancer_getdaily.sh
[alces@login1(gwtest) ~]$ pdsh -g nodes 'alces storage ls s3://jobscripts'
ip-10-75-0-91: alces storage list: storage type not available or not enabled: s3
ip-10-75-0-162: alces storage list: storage type not available or not enabled: s3
pdsh@login1: ip-10-75-0-91: ssh exited with exit code 1
pdsh@login1: ip-10-75-0-162: ssh exited with exit code 1
[alces@login1(gwtest) ~]$ qrsh
Warning: Permanently added '[ip-10-75-0-91.eu-west-1.compute.internal]:53904,[10.75.0.91]:53904' (ECDSA) to the list of known hosts.
<<< -[ alces flight ]- >>>
[alces@ip-10-75-0-91(gwtest) ~]$ alces storage ls
alces storage list: storage type not available or not enabled: s3
[alces@ip-10-75-0-91(gwtest) ~]$ alces storage enable s3
  >>> alces service install: service already installed: s3cmd
alces storage enable: enabled storage type: base/s3 -> s3
[alces@ip-10-75-0-91(gwtest) ~]$ alces storage ls s3://jobscripts
                       DIR   s3://jobscripts/jobdata/
                       DIR   s3://jobscripts/scripts/
2016-05-21 16:08  65536000   s3://jobscripts/.
2016-05-29 16:36    149853   s3://jobscripts/LC82030242016125LGN00.jpg
2016-05-29 16:23      1055   s3://jobscripts/tropic_cancer_getdaily.sh
[alces@ip-10-75-0-91(gwtest) ~]$
```

**Comment by mjtko** [comment]
Refs alces-software/clusterware-handlers#21

**Comment by mjtko** [comment]
Fixed in alces-software/clusterware-handlers#21

**Comment by vlj91** [comment]
This is included in `2016.3`
